packages/core/src/components/tabs/tabTitle.tsx

```diff
@@ -13,13 +13,13 @@ import { ITabProps, TabId } from "./tab";
 
 export interface ITabTitleProps extends ITabProps {
     /** Handler invoked when this tab is clicked. */
-    onClick: (id: TabId, event: React.MouseEvent<HTMLElement>) => void;
+    onClick?: (id: TabId, event: React.MouseEvent<HTMLElement>) => void;
 
     /** ID of the parent `Tabs` to which this tab belongs. Used to generate ID for ARIA attributes. */
-    parentId: TabId;
+    parentId?: TabId;
 
     /** Whether the tab is currently selected. */
-    selected: boolean;
+    selected?: boolean;
```

Comment:
why make these optional? would prefer to not change this component, as it's internal-only.